.gitignore

```diff
@@ -62,3 +62,4 @@ typings/
 
 example/local-data
 example/dev-bundle
+example/bundle
```

Comment:
We should remove this - this is necessary for the examples build process since this will get committed and pushed to a separate branch.